src/components/containers/TraceAccordion.js

```diff
@@ -24,14 +24,36 @@ class TraceAccordion extends Component {
     // we don't want to include analysis transforms when we're in the create panel
     const base = props.canGroup ? context.fullData : context.data;
 
-    this.filteredTracesFullDataPositions = [];
-    this.filteredTraces = base.filter((t, i) => {
-      if (props.excludeFits) {
-        return !(t.transforms && t.transforms.every(tr => tr.type === 'fit'));
+    this.filteredTracesFullDataPositions = base.map((t, i) => {
+      if (
+        props.excludeFits &&
```

Comment:
OK we have three filters here... can we just pass in one filter function and have the filter code defined at the call point rather than have this big if?